addons/l10n_be_edi/models/account_edi_format.py

```diff
@@ -21,12 +22,12 @@ def _get_efff_values(self, invoice):
     def _export_efff(self, invoice):
         self.ensure_one()
         # Create file content.
-        xml_content = b"<?xml version='1.0' encoding='UTF-8'?>"
+        xml_content = markupsafe.Markup("<?xml version='1.0' encoding='UTF-8'?>")
```

Comment:
useful? the result is stored in an attachment, and will be retrieved as b64 or bytes, so whatever Markup is generated here will be discarded immediately, no?